app/models/item.rb

```diff
@@ -7,9 +7,9 @@ class Item <ApplicationRecord
   validates_presence_of :name,
                         :description,
                         :price,
-                        :image,
                         :inventory
   validates_numericality_of :price, greater_than: 0
+  validates_numericality_of :inventory, greater_than: 0, on: :item_inventory
```

Comment:
This only validates greater than 0 when item.valid?(:item_inventory) is called. Seems very useful for future use